### PR TITLE
vim@7.4: depend on perl, python and ruby instead of :requirements

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/vim@7.4.rb [hunk 1]
```diff
@@ -18,13 +18,7 @@ class VimAT74 < Formula
...
 
   LANGUAGES_OPTIONAL = %w[lua mzscheme python3 tcl].freeze
-  LANGUAGES_DEFAULT  = %w[perl python ruby].freeze
-
-  if MacOS.version >= :mavericks
-    option "with-custom-python", "Build with a custom Python 2 instead of the Homebrew version."
-    option "with-custom-ruby", "Build with a custom Ruby instead of the Homebrew version."
-    option "with-custom-perl", "Build with a custom Perl instead of the Homebrew version."
-  end
+  LANGUAGES_DEFAULT  = %w[python].freeze
```
Comment (DomT4) [inline Formula/vim@7.4.rb]: Why the change here?

Comment (ilovezfs) [reply Formula/vim@7.4.rb]: mirroring `vim`

Comment (DomT4) [reply Formula/vim@7.4.rb]: Ah, missed that tweak! Thanks.

**Comment by ilovezfs** [comment]
:rocket: :boom: :zap:
